.github/workflows/main.yml

```diff
@@ -58,3 +58,50 @@ jobs:
         with:
           name: Win64
           path: build/*.zip
+
+  msys2:
+    runs-on: windows-latest
+
+    defaults:
+      run:
+        shell: msys2 {0}
+
+    strategy:
+      matrix:
+        include:
+          - { sys: mingw-w64, env: x86_64 }
+          - { sys: mingw-w64, env: i686 }
+
+    steps:
+      - name: Setup MSYS2
+        uses: msys2/setup-msys2@v2
+        with:
+          update: true
+          install: >-
+            git
+            ${{ matrix.sys }}-${{ matrix.env }}-gcc
+            ${{ matrix.sys }}-${{ matrix.env }}-pkgconf
+            ${{ matrix.sys }}-${{ matrix.env }}-ninja
+            ${{ matrix.sys }}-${{ matrix.env }}-cmake
+            ${{ matrix.sys }}-${{ matrix.env }}-SDL2
+            ${{ matrix.sys }}-${{ matrix.env }}-SDL2_mixer
+            ${{ matrix.sys }}-${{ matrix.env }}-SDL2_net
+
+      - uses: actions/checkout@v2
+
+      - name: Configure
+        run: cmake -B build -G "Ninja"
```

Comment:
Can we do release builds with MSYS?

Reply:
Yes, I'm trying to figure it out. Not as beautiful as in the example of course 😄